setup.sh

```diff
@@ -61,9 +61,19 @@ then
   ${DFX} canister create internet_identity --network "${NETWORK}" --no-wallet
 fi
 
-${DFX} canister create nns-dapp --network "${NETWORK}" --no-wallet
+if [ "${TESTNET}" == "local" ]
+then
+  ${DFX} canister create nns-dapp --network "${NETWORK}" --no-wallet --specified-id qsgjb-riaaa-aaaaa-aaaga-cai
+else
+  ${DFX} canister create nns-dapp --network "${NETWORK}" --no-wallet
+fi
 
-${DFX} --provisional-create-canister-effective-canister-id 5v3p4-iyaaa-aaaaa-qaaaa-cai canister create sns_aggregator --network "${NETWORK}" --no-wallet
+if [ "${TESTNET}" == "local" ]
+then
+  ${DFX} canister create sns_aggregator --network "${NETWORK}" --no-wallet --specified-id qvhpv-4qaaa-aaaaa-aaagq-cai
+else
+  ${DFX} --provisional-create-canister-effective-canister-id 5v3p4-iyaaa-aaaaa-qaaaa-cai canister create sns_aggregator --network "${NETWORK}" --no-wallet
```

Comment:
Why is this option `--provisional-create-canister-effective-canister-id` not needed when installing the canisters onto a local network?

Reply:
Because this flag determines the subnet on which we create the canister (by providing a canister ID from the subnet's canister ranges) and locally there's just a single subnet.

Reply:
```suggestion
  # Supply an additional flag to determine the subnet on which we create the canister
  ${DFX} --provisional-create-canister-effective-canister-id 5v3p4-iyaaa-aaaaa-qaaaa-cai canister create sns_aggregator --network "${NETWORK}" --no-wallet
```